document/core/valid/instructions.rst

```diff
@@ -1186,13 +1186,15 @@ Memory Instructions
 
 * The memory :math:`C.\CMEMS[0]` must be defined in the context.
 
-* Then the instruction is valid with type :math:`[\I32~\I32~\I32] \to []`.
+* Let :math:`it~\limits` be the :ref:`memory type <syntax-memtype>` :math:`C.\CMEMS[0]`.
```

Comment:
Should use idxtype here here rather than the `it` abbreviation?

Reply:
it is fine, just make sure to wrap it in `\X{...}` to have produce Latex type setting.